dbt/adapters/spark/python_submissions.py

```diff
@@ -0,0 +1,282 @@
+import base64
+import time
+import requests
+from typing import Any, Dict
+import uuid
+
+import dbt.exceptions
+
+DEFAULT_POLLING_INTERVAL = 3
+
+
+class BasePythonJobHelper:
```

Comment:
To avoid getting these massive .py files how do folks feel about putting classes in separate files?

Reply:
You mean base class in one place then the ones inheritance it in separate ones? I don't really feel like a 300 line file is massive. Is there any advantage of breaking it down to 3 100 line file?

Reply:
oh definitely not massive as of now but if the logic in these classes grows we this could get large quite quickly

Reply:
Make sense! But the logic here will likely stay the same, and we probably looking at refactoring it into multi adapter format in the longer term, or starting to adopt `dbt-databricks` for databricks specific submission. So I am inclined to leave it as is for now to avoid over-optimizing.